Makefile

```diff
@@ -57,6 +61,77 @@ ifeq (SIGN, 1)
 endif
 	@echo "ZREPL RELEASE ARTIFACTS AVAILABLE IN artifacts/release"
 
+rpm: $(ARTIFACTDIR) # artifacts/_zrepl.zsh_completion artifacts/bash_completion docs zrepl-bin
```

Comment:
Not quite sure how this is working as a `make rpm` in a clean repository should fail due to missing the zrepl binary. Or am I missing something here?

Reply:
Nope, the general problem with the Makefile is that most of the targets (including zrepl-bin, for example) are phony (i.e. execute their commands every time). Very much so in this section of the makefile that packages up output from other make targets. Look at the `release` target for example, it's basically a shell script. I think of `rpm` and `deb` in a similar way.

This situation is not great but I can't find a maintainable solution to this ATM.